rskj-core/src/main/java/co/rsk/core/bc/SelectionRule.java

```diff
@@ -0,0 +1,50 @@
+package co.rsk.core.bc;
+
+import co.rsk.remasc.Sibling;
+import org.ethereum.core.Block;
+import org.ethereum.core.BlockHeader;
+import org.ethereum.util.FastByteComparisons;
+
+import java.math.BigInteger;
+import java.util.List;
+
+public class SelectionRule {
+
+    public static boolean shouldWeAddThisBlock(BigInteger blockDifficulty, BigInteger currentDifficulty,
```

Comment:
I'm fine with all the magic numbers as long as they're contained in this class. But please add some constants to make clear what they mean.